explorer/e2e/testFunctions.ts

```diff
@@ -522,4 +541,255 @@ export async function testClearAll(
     await page.locator("body").click();
   }
 }
+
+const getRowLocatorByAccess = (page: Page, access: string): Locator =>
+  page
+    .getByRole("row")
+    .filter({ has: page.getByRole("cell", { name: access }) })
+    .first()
+    .getByRole("cell")
+    .first()
+    .getByRole("link");
+
+// Backpages tests
+export async function testExportBackpage(
+  context: BrowserContext,
+  page: Page,
+  tab: TabDescription
+): Promise<void> {
+  if (tab.backpageExportButtons == null || tab.backpageAccessTags == null) {
+    // Fail if this test is ran on a tab without defined backpages
+    await expect(false);
+    return;
+  }
+  // Goto the specified tab
+  await page.goto(tab.url);
+  // Expect to find row with a granted status indicator
+  const grantedRowLocator = getRowLocatorByAccess(
+    page,
+    tab.backpageAccessTags.grantedShortName
+  );
+  await expect(grantedRowLocator).toBeVisible();
+  // Click into the selected row
+  await grantedRowLocator.dispatchEvent("click");
+  await expect(
+    page.getByText(tab.backpageExportButtons.detailsName)
+  ).toBeVisible();
+  // Click the "Export" tab
+  await page
+    .getByText(tab.backpageExportButtons.exportTabName, {
+      exact: true,
+    })
+    .click();
+  await expect(page).toHaveURL(tab.backpageExportButtons.exportUrlRegExp);
+  await expect(page.getByRole("checkbox").first()).toBeVisible();
+  const firstButtonLocator = page.getByRole("button", {
+    name: tab.backpageExportButtons.firstButtonName,
+  });
+  await expect(firstButtonLocator).toBeEnabled();
+  // Select all checkboxes on the pages
+  const checkboxLocators = await page.getByRole("checkbox").all();
```

Comment:
Yep - the code is written so that this checks all checkboxes, and then unchecks them so that each table only has one table. That made the most sense to me when I was writing, but it can probably be done better, I can look into other ways.